knowledge-base/grid-pre-select-checkbox-filter.md

```diff
@@ -0,0 +1,83 @@
+---
+title: How can I pre-set a checkbox filter using a filter descriptor?
+description: 
+type: how-to
+page_title: How can I pre-set a checkbox filter using a filter descriptor?
+slug: grid-pre-select-checkbox-filter
+position: 
+tags: grid,checkbox,filter
+ticketid: 1541980
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Grid for Blazor</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+How can I pre-set a checkbox filter using a filter descriptor? How to set checkbox filter on Grid first load? How can I statically specify the checkbox list content? How to set checkbox filter on the first initialization of the Grid?
+
+## Solution
```

Comment:
The solution is missing several steps - 
* creating a new GridState instance
* adding the new CompositeFilterDescriptor instance to the `FilterDescriptors` collection in the state
* applying the new state to the current Grid state

Reply:
Done.